config.yml

```diff
@@ -338,6 +338,52 @@ nodes:
 
           [1, 2, 3]
           ^^^^^^^^^
+  - name: HashNode
+    child_nodes:
+      - name: opening
+        type: token
+      - name: elements
+        type: node[]
+      - name: closing
+        type: token
+    location: opening->closing
+    comment: |
+      Represents an hash literal
+
+          { a => b }
+          ^^^^^^^^^^
+  - name: AssocNode
+    child_nodes:
+      - name: opening
```

Comment:
Ahh I see how you would make this inference, but we don't need these tokens here. You can drop the `opening` and `closing` tokens here, and just use the `key->value` syntax for the location. That should simplify the tree.